next.clayui.com/src/components/clay/Input.js

```diff
@@ -0,0 +1,234 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm, {ClayInput} from '@clayui/form';
+import React from 'react';
+
+import Editor from './Editor';
+
+const spritemap = '/images/icons/icons.svg';
+
+const formInputGroupImports = `import ClayForm, {ClayInput} from '@clayui/form';
```

Comment:
Ideally each snippet should have its import, so there are some cases that some snippets don't have all these imports. You can reuse it in places that need the same import.